src/components/Checkbox/index.tsx

```diff
@@ -46,24 +56,35 @@ const StyledIcon = styled(Icon)`
 `
 
 const StyledChildrenContainer = styled('div', {
-  shouldForwardProp: prop => !['size'].includes(prop),
+  shouldForwardProp: prop => !['size'].includes(prop.toString()),
 })``
 
-const Checkbox = ({
-  checked,
-  onChange,
-  onFocus,
-  onBlur,
+type CheckboxProps = {
+  children: ReactNode
+  valid: boolean | undefined
+  error: string | ReactNode
```

Comment:
Same for error, in propTypes it is not required but here it is

Reply:
true i stick to proptypes